### ISSUE TITLE
not compatible with R 3.4.1?

### ISSUE THREAD

**Issue by markjohnsonubc** [USGS-R/Rainmaker#24]
> install.packages("Rainmaker")
Warning in install.packages :
  package ‘Rainmaker’ is not available (for R version 3.4.1)

I'm not sure what changes are going to be included in 3.4.2, but that update looks to be released on Sept 28 2017, though RC is available for developers now - 
Thursday 2017-09-21: CODE FREEZE (3.4.2 RC)

**Comment by ldecicco-USGS** [comment]
My guess is you were trying to install from CRAN. You either need to add "GRAN" (the USGS R-repository) to your profile (instructions are in the readme), or add GRAN to the repo argument:


```
install.packages("Rainmaker", repos=c("https://owi.usgs.gov/R",getOption("repos")))

```

**Comment by ldecicco-USGS** [comment]
Ha, I lied! Sorry! `Rainmaker` has not been added to the GRAN server. The maintainer makes that decision, in the meantime, you'll need to install via `devtools`:

```
devtools::install_github("USGS-R/Rainmaker")
```

**Comment by markjohnsonubc** [comment]
Thanks for the suggestion, but I had followed the instructions for adding the GRAN to my profile as per the readme. 

Executing the line with GRAN in the repo argument throws the same error:
"Warning in install.packages :
  package ‘Rainmaker’ is not available (for R version 3.4.1)"

Should it be working under R version 3.4.1?

**Comment by markjohnsonubc** [comment]
Oops - our comments crossed. 

I've installed via devtools as you suggest. All good now.
